src/main/resources/db/migration/R__grant_to_cloudsqliamuser.sql

```diff
@@ -0,0 +1 @@
+GRANT SELECT ON ALL TABLES IN SCHEMA public TO cloudsqliamuser;
```

Comment:
Tilsvarende for "isyfo-analyse"

Reply:
Oppdatert migration med grant til Isyfo-analyse.